Update faq-md-app-guard.md (#8973)
MicrosoftDocs/windows-itpro-docs: windows/security/threat-protection/microsoft-defender-application-guard/faq-md-app-guard.md

5. Reboot the device.

> ### Why doesn't the container not fully load when Device Control Policies are enabled?

Reviewer: ```suggestion
### Why doesn't the container fully load when device control policies are enabled?
```

The whitelisting of these items are required to be allowed in the GPO to ensure AppGuard works properly. 

> Policy: Allow installation of devices that match any of these device IDs 
> •	SCSI\DiskMsft____Virtual_Disk____ 
> •	{8e7bd593-6e6c-4c52-86a6-77175494dd8e}\msvhdhba 
> •	VMS_VSF 
> •	root\Vpcivsp 
> •	root\VMBus 
> •	vms_mp 
> •	VMS_VSP 
> •	ROOT\VKRNLINTVSP 
> •	ROOT\VID 
> •	root\storvsp 
> •	vms_vsmp 
> •	VMS_PP 
> 
> Policy: Allow installation of devices using drivers that match these device setup classes 
> •	{71a27cdd-812a-11d0-bec7-08002be2092f}

Reviewer 2: This bullet point list is still broken, because it contains pre-formatted characters and tabs, which does not belong in MarkDown code. Please add my suggested change below to unbreak your PR content:
```suggestion
Policy: Allow installation of devices that match any of these device IDs 
- SCSI\DiskMsft____Virtual_Disk____ 
- {8e7bd593-6e6c-4c52-86a6-77175494dd8e}\msvhdhba 
- VMS_VSF 
- root\Vpcivsp 
- root\VMBus 
- vms_mp 
- VMS_VSP 
- ROOT\VKRNLINTVSP 
- ROOT\VID 
- root\storvsp 
- vms_vsmp 
- VMS_PP 

Policy: Allow installation of devices using drivers that match these device setup classes 
- {71a27cdd-812a-11d0-bec7-08002be2092f}
```

> ### Why doesn't the container not fully load when Device Control Policies are enabled?
> The whitelisting of these items are required to be allowed in the GPO to ensure AppGuard works properly. 
> 
> Policy: Allow installation of devices that match any of these device IDs 
> •	SCSI\DiskMsft____Virtual_Disk____ 
> •	{8e7bd593-6e6c-4c52-86a6-77175494dd8e}\msvhdhba 
> •	VMS_VSF 
> •	root\Vpcivsp 
> •	root\VMBus 
> •	vms_mp 
> •	VMS_VSP 
> •	ROOT\VKRNLINTVSP 
> •	ROOT\VID 
> •	root\storvsp 
> •	vms_vsmp 
> •	VMS_PP 
> 
> Policy: Allow installation of devices using drivers that match these device setup classes 
> •	{71a27cdd-812a-11d0-bec7-08002be2092f}
> 

Reviewer 2: Allow me to suggest some improvements, as well as a couple of troubleshooting fixes:

1. Avoid using TAB in a bullet point list or numbered list. Tab can break the MarkDown formatting.
2. Only 1 space is needed in bullet point lists and numbered lists.
3. Replicating the Rich Diff view spacing in source view by adding single blank lines after headings is often useful to notice if something needs improving.

```suggestion
### Why does the container not fully load when Device Control Policies are enabled?

Whitelisting the following items is required and necessary to be allowed in the GPO, to ensure that AppGuard works properly:

Policy: Allow installation of devices that match any of these device IDs

• SCSI\DiskMsft____Virtual_Disk____
• {8e7bd593-6e6c-4c52-86a6-77175494dd8e}\msvhdhba
• VMS_VSF
• root\Vpcivsp
• root\VMBus
• vms_mp
• VMS_VSP
• ROOT\VKRNLINTVSP
• ROOT\VID
• root\storvsp
• vms_vsmp
• VMS_PP

Policy: Allow installation of devices using drivers that match these device setup classes

• {71a27cdd-812a-11d0-bec7-08002be2092f}
```

Reviewer 2: Please note that I have only added extra blank lines where they would otherwise appear as a result of MarkDown formatting in Rich Diff view and the resulting HTML page on docs.microsoft.com . Thank you.
